### PR TITLE
Resizing of the logo on the README.md

### PR DESCRIPTION
**Pull request by skmu104**
**Description**

Re-sizing of the logo on the README so it doesn't look out of place

**Checklist**

- [x] Docs have been added/updated if needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Split
-![Split Logo](https://i.imgur.com/qXONyae.jpg)
+<img src="https://i.imgur.com/qXONyae.jpg" width="400" height="250" />
```
Comment (williamlixu) [inline README.md]: Resizing the logo like this kinda changes the resolution and makes it look a bit squished. Would prefer if the image was cropped and reuploaded!

**Review by williamlixu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (skmu104) [reply README.md]: Could either @SamNeale or @michellemtan look into cropping and reuploading the image Thanks

Comment (Curtiss02) [reply README.md]:
``` suggestion
![split](https://user-images.githubusercontent.com/8885794/77110524-91a8a080-6a8a-11ea-9ad0-950b17ac6c93.png)
```
Maybe something like this. I just eyeballed it so if someone actually has the source files feel free to do it properly.

**Review by Curtiss02 — CHANGES_REQUESTED** [review]

**Review by williamlixu — APPROVED** [review]
Thanks for this, looks great!

**Review by Curtiss02 — APPROVED** [review]
lgtm

**Pull request merged**